commit/merkleroot/outcome.go

```diff
@@ -125,12 +127,31 @@ func buildReport(
 
 	sort.Slice(roots, func(i, j int) bool { return roots[i].ChainSel < roots[j].ChainSel })
 
-	// TODO: use q.RMNSignatures in the generated outcome and eventually report. - Blocked by onchain work.
+	sigs := make([]cciptypes.RMNECDSASignature, 0)
+	if q.RMNSignatures == nil {
+		lggr.Error("RMNSignatures are nil")
+	} else {
+		parsedSigs, err := rmn.NewECDSASigsFromPB(q.RMNSignatures.Signatures)
+		if err != nil {
+			lggr.Errorw("Failed to parse RMN signatures", "error", err)
+		} else {
+			sigs = parsedSigs
+		}
+	}
+	sort.Slice(sigs, func(i, j int) bool {
+		riStr := sigs[i].R.String()
+		rjStr := sigs[j].R.String()
+		if riStr != rjStr {
+			return riStr < rjStr
+		}
+		return sigs[i].S.String() < sigs[j].S.String()
```

Comment:
Do we want to do a lexicographical sort? I thought the sort had to be on the recovered pubkey / address.

Reply:
c.f https://github.com/smartcontractkit/chainlink/blob/a2514d725f66e2c4a66d9bd0dd211a617fa32367/contracts/src/v0.8/ccip/rmn/RMNRemote.sol#L101

Reply:
nice catch

Reply:
fixed and added a test